supertokens_python/recipe/emailverification/recipe.py

```diff
@@ -313,9 +314,7 @@ async def fetch_value(user_id: str, user_context: Dict[str, Any]) -> bool:
             if isinstance(email_info, EmailDoesNotExistError):
                 # we consider people without email addresses as validated
                 return True
-            raise Exception(
-                "Should never come here: UNKNOWN_USER_ID or invalid result from get_email_for_user"
-            )
+            raise_unauthorised_exception("UNKNOWN_USER_ID")  # todo: empty in node SDK
```

Comment:
- As per the node PR, this should throw a regular exception with the UNKNOWN_USER_ID message
- Why is there a TODO next to it? Is something wrong with the node SDK?